Source/WebKit/Configurations/AllowedSPI.toml

```
@@ -241,4 +247,6 @@ classes = [
 selectors = [
     { name = "buildMenuOfType:", class = "AVLegibleMediaOptionsMenuController" },
 ]
-requires = ["HAVE_AVLEGIBLEMEDIAOPTIONSMENUCONTROLLER"]
+requires = ["HAVE_AVLEGIBLEMEDIAOPTIONSMENUCONTROLLER"
+
+]
```

nit: looks like noise

```suggestion
requires = ["HAVE_AVLEGIBLEMEDIAOPTIONSMENUCONTROLLER"]
```